files/grafana/nginx-status.conf

```diff
@@ -0,0 +1,9 @@
+server {
```

Comment:
I think you don't have an `nginx` server? So this is not needed.

Reply:
well don't i need a webserver to see this stuff?

Reply:
Yes, but this monitors your nginx server in that case, its the performance of that webserver (so less relevant)

Reply:
removed